client/my-sites/stats/activity-log/threat-alert.jsx

```diff
@@ -4,7 +4,7 @@
  */
 import React, { Component, Fragment } from 'react';
 import { localize } from 'i18n-calypso';
-
+import debugFactory from 'debug';
```

Comment:
Nit: I'd probably prefer keeping that empty line below the import.

Reply:
Good catch, hadn't seen that one!